controllers/list/submissions/PKPSubmissionsListHandler.inc.php

```diff
@@ -104,6 +104,11 @@ public function getConfig() {
 						'val' => true,
 						'title' => __('submissions.incomplete'),
 					),
+					array(
+						'param' => 'isInactive',
+						'val' => true,
```

Comment:
So here is where, once we're using an `int`, you can pass in `30` as the set value for the UI. This is only temporary but will set us up properly for making it adjustable later.